src/similar-npm-packages.ts

```diff
@@ -567,6 +597,16 @@ export const categories: Categories = {
     similar: ["raphael", "snapsvg", "two.js"],
   },
 
+  "template-engine": {
+    name: "Template engines",
+    tags: [
+      { tag: "template", weight: Weight.MAX },
+      { tag: "partial", weight: Weight.HIGH },
+      { tag: "html", weight: Weight.NORMAL },
+    ],
+    similar: ["handlebars", "eta"],
```

Comment:
You might want to include —

`nunjucks` (even though it's quite heavy, it's very popular and good for a reference)
`ejs`(relatively light, widely used)
`dot` (lightweight, fast, popular).

`eta` seems to be new to the scene, and has yet to gain wider community adoption. 
Do you think it deserves promotion over others in this list?

Reply:
👍 Added all packages you suggested.
`eta` looks good. We have a place in this category so I would keep it.